service.subtitles.legendasdivx/resources/language/Portuguese/strings.po

```diff
@@ -0,0 +1,122 @@
+# Addon Name: LegendasDivx.com
```

Comment:
move this file to the new language folder layout:

`resources/language/resource.language.pt_pt/strings.po`

Reply:
ok